client/components/premium-global-styles-upgrade-modal/index.tsx

```diff
@@ -40,23 +38,12 @@ export default function PremiumGlobalStylesUpgradeModal( {
 		translate( 'Ad-free experience' ),
 		translate( 'Earn with WordAds' ),
 	];
-	const personalFeatures = [
-		<strong>{ translate( 'Free domain for one year' ) }</strong>,
-		<strong>{ translate( 'Style customization' ) }</strong>,
-		translate( 'Support via email' ),
-		translate( 'Ad-free experience' ),
-	];
-	const displayFeatures = globalStylesInPersonalPlan ? personalFeatures : features;
```

Comment:
I know this is a bit out of scope and we can do this as a follow-up. 
Should we add the `Style Customization` feature to the features list of the `Premium plan`?

Reply:
I think that's the case already: 

https://github.com/Automattic/wp-calypso/blob/291a55ae1fe9bf6357649579c24e8828cd6966b5/client/components/premium-global-styles-upgrade-modal/index.tsx#L36